.github/workflows/build.yml

```diff
@@ -69,6 +69,38 @@ jobs:
           path: |
             build/coreboot.rom
           retention-days: 30
+  build_optiplex:
+    runs-on: ubuntu-22.04
+    container:
+      image: coreboot/coreboot-sdk:2023-11-24_2731fa619b
+      options: --user 1001
+    strategy:
+      matrix:
+        vendor: [ dell ]
+        model: [ optiplex_9010_sff_uefi_txt ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # Checkout pull request HEAD commit instead of merge commit
+          # See: https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+          ref: ${{ github.event.pull_request.head.sha }}
+          # Fetch complete history
+          fetch-depth: 0
+      - name: Checkout all submodules
+        run: git submodule update --init --recursive --checkout
+      - name: Build Dasharo
+        run: |
+          cp configs/config.${{ matrix.vendor }}_${{ matrix.model }} .config
```

Comment:
```suggestion
          cp configs/config.${{ matrix.vendor }}_${{ matrix.model }}_uefi_txt .config
```

Reply:
Changed in [this commit](https://github.com/Dasharo/coreboot/pull/541/commits/0a7ec66884e093d282af3e56760359b4c0fc14bf)